R/add_meta.R

```diff
@@ -19,7 +19,10 @@
 #' @seealso \code{\link{add_contributing}}, \code{\link{add_license}}, \code{\link{add_license_header}}
 #' @return NULL
 add_readme <- function(path = ".", package = FALSE, CoC = TRUE) {
+  if (CoC == TRUE) add_code_of_conduct()
```

Comment:
That was a useful little argument there that wouldn't actually do anything. Thanks!